dbt/models/staging/staging.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
```

Comment:
In general wondering how these tests are?

Reply:
Looks great overall! This looks nice and robust.

Once idea: think about adding a few checks for the logic instead of `analytics_flagged`. There are a few complex calculations in there that might silently break if problems exist!

Reply:
We discussed this a bit offline, but adding in here! Ideally, you want to keep the error as a "reminder" until things are fixed at the source. That doesn't always work, so sometimes you need to remove the error to make sure you aren't adding too much noise